src/GLUniforms.jl

```diff
@@ -142,6 +145,7 @@ gl_promote(x::Type{Union{UInt16, UInt8}})  = x
 
 gl_promote{T <: AbstractFloat}(x::Type{T}) = Float32
 gl_promote(x::Type{Float16})               = x
+gl_promote(x::Type{Float64})               = x
```

Comment:
Ah, that would make sense. I need Double support for some things, and just went to everywhere that called for GLfloats and Float32s. I will go make that change. Thanks for the feedback.